src/core/server/plugins/plugin.ts

```diff
@@ -237,6 +242,23 @@ export class PluginWrapper<
     this.instance = undefined;
   }
 
+  public getConfigSchema(): PluginConfigSchema {
+    if (!this.manifest.server) return null;
+    const pluginPathServer = join(this.path, 'server');
```

Comment:
__optional nit:__ would you mind adding a new line after `if`? Just to make it more visible....